spec/openapi3.yaml

```diff
@@ -3165,6 +3263,99 @@ components:
       allOf:
         - $ref: "#/components/schemas/EmbeddedTransactionDTO"
         - $ref: "#/components/schemas/MosaicGlobalRestrictionTransactionBodyDTO"
+    MosaicGlobalRestrictionEntryRestrictionDTO:
+      type: object
+      required:
+        - referenceMosaicId
+        - restrictionValue
+        - restrictionType
+      properties:
+        referenceMosaicId:
+          $ref: "#/components/schemas/MosaicId"
+        restrictionValue:
+          type: string
+          description: Restriction value.
+        restrictionType:
+          type: integer
+    MosaicGlobalRestrictionEntryDTO:
+      type: object
+      required:
+        - key
+        - restriction
+      properties:
+        key:
+          type: string
+          description: Restriction key.
+          example: 0DC67FBE1CAD29E3
+        restriction:
+          $ref: "#/components/schemas/MosaicGlobalRestrictionEntryRestrictionDTO"
+    MosaicGlobalRestrictionEntryWrapperDTO:
+      type: object
+      required:
+        - compositeHash
+        - entryType
+        - mosaicId
+        - restrictions
+      properties:
+        compositeHash:
+          $ref: "#/components/schemas/Hash256"
+        entryType:
+          $ref: "#/components/schemas/MosaicRestrictionEntryTypeEnum"
+        mosaicId:
+          $ref: "#/components/schemas/MosaicId"
+        restrictions:
+          type: array
+          items:
+            $ref: "#/components/schemas/MosaicGlobalRestrictionEntryDTO"
+    MosaicGlobalRestrictionDTO:
+      type: object
+      required:
+        - mosaicRestrictionEntry
+      properties:
+        mosaicRestrictionEntry:
+          $ref: "#/components/schemas/MosaicGlobalRestrictionEntryWrapperDTO"
+    MosaicAddressRestrictionEntryDTO:
+      type: object
+      required:
+        - key
+        - value
+      properties:
+        key:
+          type: string
+          description: Restriction key.
```

Comment:
format: hex

Reply:
We can also abstract the MosaicRestrictionKey as a separate object as we are using it in multiple places.